publish/publish.proj

```diff
@@ -69,14 +69,14 @@
     <FinalizeBuild AccountName="$(AzureAccountName)"
                    AccountKey="$(AzureAccessToken)"
                    ContainerName="$(ContainerName)"
-                   SemaphoreBlob="$(Channel)/Binaries/sharedFxPublishSemaphore"
+                   SemaphoreBlob="Runtime/sharedFxPublishSemaphore"
```

Comment:
The semaphore should be inside the channel.  So `Runtime/$(Channel)/sharedFxPublishSemaphore`.

If 2 builds from separate channels (branches) are publishing at the same time, there is no need to control the concurrency between them.  But if 2 builds in the same channel are publishing at the same time, we need to control the concurrency so they both don't write to the channel folder at the same time.